### PR TITLE
Implemented and tested Rot90 for Paddle frontends

### PR DESCRIPTION
**Pull request by XinyuanWang283**
Close#16746

### CODE REVIEW

**Comment by XinyuanWang283** [comment]
Close #16746

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by XinyuanWang283** [comment]
Hi! @DragosStoican Could you please help me check about my code?

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 1]
```diff
@@ -1285,3 +1288,45 @@ def test_paddle_bitwise_xor(
...
     )
+
+
+# rot90
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="rot90",
+    dtype_m_k_axes=_get_dtype_values_k_axes_for_rot90(
+        available_dtypes=helpers.get_dtypes("numeric"),
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1299]: We should be ideally testing for all valid dtypes, and not only a subset.

Comment (XinyuanWang283) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1299]: The numeric has included  float, integer, bool types that are all supported by Rot90.  you can check https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex. Please free free to ask me any questions. Thank you!

Comment (XinyuanWang283) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1299]: @DragosStoican

Comment (XinyuanWang283) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py]: Thank you very much for your guidance. @DragosStoican  Sorry, I made a mistake. I have fixed it. Could you help me review it again. Thanks!

**Comment by DragosStoican** [comment]
Thank you very much for your contribution and cooperation @XinyuanWang283. I'm happy to get this merged now!

**Pull request merged**